components/Common/ThemeToggle/index.tsx

```diff
@@ -11,7 +11,7 @@ type ThemeToggleProps = {
 const ThemeToggle: FC<ThemeToggleProps> = ({ onClick = () => {} }) => {
   const t = useTranslations();
 
-  const ariaLabel = t('components.header.buttons.toggleDarkMode');
+  const ariaLabel = t('components.header.buttons.toggleTheme');
```

Comment:
Since it's possible to toggle theme in both directions (light -> dark and dark -> light), IMHO `toggleTheme` is more relevant for the logic. SImilar to other names changes across PR